.fixtures.yml

```diff
@@ -5,6 +5,6 @@ fixtures:
       ref: '5.2.0'
     concat:
       repo: "https://github.com/puppetlabs/puppetlabs-concat.git"
-      ref: '1.2.1'
+      ref: '6.4.0'
```

Comment:
this should be `v6.4.0` :)

https://github.com/puppetlabs/puppetlabs-concat/releases

Reply:
Thanks for fixing that.